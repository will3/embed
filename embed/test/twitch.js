const test = require('./utils/test');

test('twitch',
  'https://www.twitch.tv/speedgaming');
