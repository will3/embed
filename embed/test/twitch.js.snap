// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` twitch 1`] = `
Object {
  "description": undefined,
  "originalUrl": "https://www.twitch.tv/speedgaming",
  "site_name": "Twitch",
  "thumbnail": Object {
    "url": "https://static-cdn.jtvnw.net/jtv_user_pictures/speedgaming-profile_image-b15b320868ba3342-300x300.png",
  },
  "title": "Twitch",
  "video": Object {
    "description": undefined,
    "height": 0,
    "title": undefined,
    "url": "http://player.twitch.tv/?channel=speedgaming&player=facebook&autoplay=true",
    "width": 0,
  },
}
`;
